client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -345,6 +348,7 @@ export const ApplicationsTable: React.FC = () => {
       businessService: app.businessService?.name || "",
       tags: app.tags?.length || 0,
       effort: app.effort || 0,
+      analysis: app.tasks.currentAnalyzer?.state || "",
```

Comment:
we should sort on labels visible on the screen -  see the propsed `mapAnalysisStateToLabel` below